assignment4/app.js

```diff
@@ -0,0 +1,26 @@
+const fs = require("fs");
+
+var fileArray = ["file1.txt", "file2.txt", "file3.txt"];
+
+function printData(err, data) {
+  if (err) console.log(err);
+  else console.log(data);
+}
+fs.readFile("file1.txt", "utf-8", function(err, data) {
```

Comment:
use [arrow function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions) instead of `function`